java-client/src/test/java11/org/opensearch/client/opensearch/integTest/AbstractMultiSearchRequestIT.java

```diff
@@ -247,6 +247,21 @@ public void shouldReturnMultiSearchesFields() throws Exception {
         assertThat(response.responses().get(0).result().hits().hits().get(2).fields(), hasKey("name"));
     }
 
+    @Test
+    public void shouldReturnMultiSearchesDocvalueFields() throws Exception {
+        String index = "multiple_searches_request_docvalue_fields";
+        createTestDocuments(index);
+
+        RequestItem sortedItemsQuery = createMSearchFuzzyRequest(b -> b.docvalueFields(FieldAndFormat.of(f -> f.field("name"))));
+
+        MsearchResponse<ShopItem> response = sendMSearchRequest(index, List.of(sortedItemsQuery));
+        assertEquals(1, response.responses().size());
+        assertEquals(3, response.responses().get(0).result().hits().hits().size());
```

Comment:
Integration tests are failing on the `.result()` call as the response is a failure for some reason

Reply:
Fixed the integration tests, essentially doc value fields are not allowed on text fields (`name`). Changing that to `quantity` fixed the issue